sdk/src/Services/DynamoDBv2/Custom/DataModel/ContextInternal.cs

```diff
@@ -359,6 +359,14 @@ private object FromDynamoDBEntry(SimplePropertyStorage propertyStorage, DynamoDB
             }
         }
         private bool TryFromList(Type targetType, DynamoDBList list, DynamoDBFlatConfig flatConfig, out object output)
+        {
```

Comment:
It can reuse the `Utils.ItemsToCollection` method which is used by `PrimitiveCollectionConverter` and `DynamoDBListConverter` here.
If you would like this, I can include this commit (https://github.com/jiabiao/aws-sdk-net/pull/1/commits/32543b36e750fa654ef9dbadb55c72c2fe64bb56) in this PR.